### PR TITLE
fix: [UI] fix cursor still blinks in the input box

### PR DESCRIPTION
**Pull request by add-uos**
fix: [UI] fix cursor still blinks in the input box

Change the user name after pressing enter, the cursor still blinks in the input box. We fixed.

Log: Change the user name after pressing enter, the cursor still blinks in the input box.
Bug: https://pms.uniontech.com/bug-view-241037.html

### CODE REVIEW

File: src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.cpp [hunk 1]
```diff
@@ -446,12 +451,16 @@ void FirstTipWidget::themeTypeChanged()
         firstTip->setStyleSheet("background-color: rgba(255, 255, 255, 0.03); "
                                 "border-radius: 10px;"
                                 "color: rgba(255, 255, 255, 0.6);");
+#ifndef linux
         tipBtn->setIcon(QIcon::fromTheme(":/icons/deepin/builtin/dark/icons/tab_close_normal.svg"));
+#endif
```
Comment (pppanghu77) [inline src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.cpp:456]: window没有深色模式，不需要适配主题切换

File: src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.cpp [hunk 2]
```diff
@@ -446,12 +451,16 @@ void FirstTipWidget::themeTypeChanged()
...
     } else {
         firstTip->setStyleSheet("background-color: white; "
                                 "border-radius: 10px;"
                                 "color: rgba(0, 0, 0, 0.6);");
-        tipBtn->setIcon(QIcon::fromTheme(":/icons/deepin/builtin/icons/close_normal.svg"));
+#ifndef linux
+        tipBtn->setIcon(QIcon::fromTheme(":/icons/deepin/builtin/icons/clear.svg"));
+#endif
```
Comment (pppanghu77) [inline src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.cpp:463]: 同上

File: src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.cpp [hunk 3]
```diff
@@ -463,21 +472,15 @@ void FirstTipWidget::initUI()
     firstTip->setWordWrap(true);
     firstTip->setFixedWidth(480);
     firstTip->setContentsMargins(10, 10, 40, 10);
-
+#ifdef linux
+    tipBtn = new DIconButton(dynamic_cast<QWidget *>(parent()));
```
Comment (pppanghu77) [inline src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.cpp:476]: 参考cooperationbtn的设计，在type-define中定义

File: src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.h [hunk 4]
```diff
@@ -101,7 +104,12 @@ class FirstTipWidget : public QWidget
 
 private:
     void initUI();
+#ifdef linux
+    DTK_WIDGET_NAMESPACE::DIconButton *tipBtn { nullptr };
```
Comment (pppanghu77) [inline src/plugins/cooperation/core/gui/widgets/cooperationstatewidget.h:108]: 同上

File: src/plugins/cooperation/core/gui/resources/resources.qrc [hunk 5]
```diff
@@ -35,5 +35,6 @@
         <file>icons/deepin/builtin/icons/tip_kvm.png</file>
         <file>icons/deepin/builtin/icons/tip_more.png</file>
         <file>icons/deepin/builtin/dark/icons/tab_close_normal.svg</file>
+        <file>icons/deepin/builtin/icons/clear.svg</file>
     </qresource>
```
Comment (pppanghu77) [inline src/plugins/cooperation/core/gui/resources/resources.qrc:39]: 这个icon原来的资源文件里面有

**Review by pppanghu77 — APPROVED** [review]

**Review by re2zero — APPROVED** [review]

**Pull request merged**